docs/sources/shared/find-and-use-plugins.md

```diff
@@ -0,0 +1,43 @@
+---
+aliases:
+  - /docs/grafana/fundamentals/find-and-use-plugins/
+  - /docs/grafana/how-do-i/find-and-use-plugins/
+  - /docs/grafana/introduction/find-and-use-plugins/
+description: Grafana Plugins
+labels:
+  products:
+    - oss
+keywords:
+  - Grafana OSS
+  - Plugins
+  - Install Plugins
+  - Download Plugins
+menuTitle: Find and use plugins
+title: Find and use plugins
+weight: 750
+---
+
+# Find and use Grafana plugins
+
+Plugins extend Grafana by adding capabilities that are not already present. There are plugins that add the ability to import data from diverse sources, that bundle data sources and panels, and that provide new visualization types for use in dashboards.
+
+To find new plugins, browse the Grafana plugins catalog which is available both [within Grafana](https://grafana.com/docs/grafana/latest/administration/plugin-management/#browse-plugins) and on the Grafana [website](/grafana/plugins/).
```

Comment:
Links should use the special syntax for version substitution so that different versions of docs don't all link to "latest".

```suggestion
To find new plugins, browse the Grafana plugins catalog which is available both [within Grafana](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/plugin-management/#browse-plugins) and on the Grafana [website](/grafana/plugins/).
```